### ISSUE TITLE
Setup CD

### ISSUE THREAD

**Issue by antonplagemann** [Music-Bot-for-Jitsi/Jimmi#25]
> **As a** developer
> **I want** my code to be compiled automatically
> **so that** a new release is instantly available as binary.

## Description:
Setup a continuous deployment pipeline on main push / release using GitHub Actions.
Depends on #59 and #60.

### 🟢 In scope:
- [x] Build and push docker containers
- [x] Build binary for Windows, Linux and MacOS
- [x] Create release and append binarys

### 🔴 Not in scope:
- Write something else beside a `.yml` file

## What should be the result?
A GitHub Actions pipeline that does the required actions automatically.

**Comment by antonplagemann** [comment]
Waiting for #59 and #60
